bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/dialogs/SearchPattern.java

```diff
@@ -85,10 +95,33 @@ public class SearchPattern {
 	 * <code>"NPE"</code> string pattern, search will only use Camel Case match
 	 * rule, but with <code>N*P*E*</code> string pattern, it will use only Pattern
 	 * match rule.
-	 *
+	 * <p>
+	 * Unless {@link #RULE_SUBSTRING_MATCH} is in effect, it is required that
+	 * the 1st pattern's char must match the very 1st char of the search result.
 	 */
 	public static final int RULE_CAMELCASE_MATCH = 0x0080;
 
+	/**
+	 * Match rule: The search pattern is a string placed anywhere in the search
+	 * result. When in effect, this flag also affects some of the other match rules
+	 * (see their documentation for details).
+	 * <p>
+	 * Prefix search may still be enforced by placing '>' at the beginning of the
+	 * pattern. Analogically, suffix search may be enforced by placing ' ' or '&lt;'
+	 * at the end of the pattern.
+	 *
+	 * @since 3.126
```

Comment:
Should now be 3.127 as 3.126 was released in the meantime.

Reply:
OK, done.